tests/release_candidate_scraper/test_release_candidate_scraper_indices.py

```diff
@@ -6,22 +6,29 @@
 
 import pytest
 
-from mozdownload import ReleaseCandidateScraper
+from mozdownload import ReleaseCandidateScraper, errors
```

Comment:
We try to keep the alphabetical order of imports. So please exchange both entries.

Reply:
ok